### PR TITLE
Fix License

### PR DESCRIPTION
**Pull request by jtaleric**
Missing date/owner.

Signed-off-by: Joe Talerico (rook) <joe.talerico@gmail.com>

### CODE REVIEW

**Review by christarazi — APPROVED** [review]

**Comment by jtaleric** [comment]
cc @learnitall

File: LICENSE [hunk 1]
```diff
@@ -175,18 +175,7 @@
...
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 Isovalent 
 
    Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (learnitall) [inline LICENSE:180]: Can we remove these two lines at the bottom? Without the Appendix they don't have any meaning.

**Review by learnitall — CHANGES_REQUESTED** [review]
I think that this woks for now, but in the future I think it would be best to have the license text verbatim, create an `AUTHORS` file, and add the following header to each source file:

```
Copyright 2022 Authors of scaffolding
SPDX-License-Identifier: Apache-2.0
```

This follows the pattern we already use at Isovalent for adding in a copyright, and will be useful for the more standalone scripts included in the repo.

File: LICENSE (hunk 1 above)
Comment (jtaleric) [reply LICENSE:180]: oh, weird. I rather add the APPENDIX back.. i got too handsy with deleting lines.

**Comment by jtaleric** [comment]
@learnitall added the appendix back, sorry for the delay didn't see this in my backlog :(

**Review by learnitall — APPROVED** [review]
LGTM

**Pull request merged**